knotx-knot/knotx-knot-fragment-assembler/src/test/java/io/knotx/assembler/FragmentAssemblerTest.java

```diff
@@ -67,7 +72,7 @@ public void callAssemblerWithNoSnippets_expectInternalServerError(TestContext co
   @KnotxConfiguration("test.unwrap.io.knotx.FragmentAssembler.json")
   public void callAssemblerWithEmptySnippet_expectNoContentStatus(TestContext context)
       throws Exception {
-    callAssemblerWithAssertions(context, Collections.singletonList(" "),
+    callAssemblerWithAssertions(context, Collections.singletonList(new ImmutablePair<>(Collections.singletonList(RAW), " ")),
```

Comment:
@tomaszmichalak TBH, `StringUtils.SPACE` is not at all better than `" "`

It's just as meaningless in this context. I'd rather introduce a constant describing its purpose. Is it a separator?